### DISCUSSION TITLE
Best way to split up different types of relays

### DISCUSSION THREAD

**Discussion by gitwormq** [nusenu/ansible-relayor#240] [Q&A]
Hello, I first want to thank you for this amazing project!
I run a handfull of exit relays but now want to also spin up some middle relays. What is the best way to accomplish this with ansible-relayor?
As an example say I want:
5 exit relays
5 middle relays
I want to ensure all 10 of these hosts are in the same family. From adding new relays in the past, I would:
1. ansible-playbook yourplaybook.yml -l newserver --tags openbsd
2. ansible-playbook yourplaybook.yml --tags reconfigure

However, this assumes you utilize the same playbook where I would want to use a different playbook for my middle relays but ensure they are all apart of the same family.
Any help would be appreciated.

**Comment by nusenu** [comment]
Hi,
your desired setup is supported out of the box.
You can simply use ansible group_vars to set the relevant variable "tor_ExitRelay" accordingly
and keep using a single playbook.
All relays will be in the same family.

kind regards,
nusenu
